app/core/api/archived-elements.js

```diff
@@ -0,0 +1,56 @@
+import fetchJson from './fetch-json'
+
+/**
+ * Archives a game element
+ * @param _id - The mongo _id of the element
+ * @param elementType - The game element type
+ * @async
+ */
+export const archiveElement = (_id, elementType) => {
```

Comment:
is this file from a different PR? I don't find usage of this function anywhere in this PR